### DISCUSSION TITLE
Add some Database for app profile | feature

### DISCUSSION THREAD

**Discussion by WUIX69** [King-Projects/King-Tweaks#6] [Ideas]
GoodAfternoon sir @pedrozzz0

Can i request for Database management for app profile, to make it a lot easier, and more conservative to battery..

Like: 
database add GAMING com.wildrift.leagueoflegends
- to avoid drain, slow respond of switching profile..

Thanks a lot, Love you 💕

**Comment by pedrozzz0** [comment]
hmm, will check it, thanks for the feedback.
